src/style.css

```diff
@@ -151,3 +151,10 @@ a {
 .token.variable {
   color: #b56000 !important; /* #e90; */
 }
+
+.token.comment,
+.token.prolog,
+.token.doctype,
+.token.cdata {
+  color: #687888 !important; /* slategray */
+}
```

Comment:
I missed these colors in the first update to the Prism colors. This time I found a workflow that had tokens from this group, and ANDI flagged the contrast again.